onebusaway-android/src/main/java/org/onebusaway/android/util/GetRestrictionsReceiver.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2013 The Android Open Source Project
```

Comment:
Do we have a source URL for where this code came from that we can add in the comment header?

Reply:
https://github.com/googlesamples/android-AppRestrictions/blob/master/Application/src/main/java/com/example/android/apprestrictions/GetRestrictionsReceiver.java

Reply:
I added this link to the comments block in https://github.com/OneBusAway/onebusaway-android/pull/744/commits/bde0c799fe9d0018e11085bdd327bb4c6623af65.